### ISSUE TITLE
Security scheme fragment doesn't use global mediaType

### ISSUE THREAD

**Issue by deiteris** [aml-org/amf#899]
Your issue may already be reported! Please search on Github issues before creating one.

* **I'm submitting a ...**
  - [x] bug report
  - [ ] feature request
  - [ ] question
  - [ ] other

* **What is the current behavior?**

With global `mediaType` defined, a response inside `securityScheme` doesn't seem to pick it up, and produces false validation error.

* **If the current behavior is a bug, please provide the minimal steps to reproduce with AMF code. Please don't upload AMF code mixed with yours.**

Here's the RAML that produces the error:

```yaml
#%RAML 1.0
title: "Pets store"
version: "v1"
protocols: 
  - HTTPS
baseUri: https://example.com/api
mediaType: application/json

securitySchemes:
  oauth_2_0:
    type: OAuth 2.0
    describedBy:
      responses:
        401:
          description: Missing or invalid token
          body:
            properties:


/pets:
  get:
    displayName: Get pets list.
    responses:
      200:
        description: Not Found
        body:
          properties:
```

The result (notice that the response in security scheme produces the issue, while in endpoint method it is not):

```
Model: amf://id
Profile: RAML 1.0
Conforms? false
Number of results: 1

Level: Violation

- Source: http://a.ml/vocabularies/amf/parser#Payload-mediaType-minCount
  Message: Payload media type is mandatory
  Level: Violation
  Target: amf://id#3
  Property: http://a.ml/vocabularies/core#mediaType
  Position: Some(LexicalInformation([(16,10)-(20,0)]))
  Location: file:///tmp/tmp-774412-3dIziJ4g8NjS
```

* **What is the expected behavior?**

No validation error, since it's in accordance with [the RAML 1.0 spec](https://github.com/raml-org/raml-spec/blob/master/versions/raml-10/raml-10.md#the-root-of-the-document).
>The root section also defines assets used elsewhere in the RAML document, such as types and traits.

* **What is the motivation/use case for changing the behavior? (for feature requests)**

* **Please tell us about your environment:**
  
  - AMF Version: 4.7.1 / 4.7.2 / 4.8.0-SNAPSHOT
  - AMF Distribution: JVM and JS
  - Operating System: Windows 10H2 / CentOS Linux release 8.3.2011

* **Other information** (e.g. detailed explanation, stack traces, related issues, suggestions how to fix, links for us to have context, eg. StackOverflow, Gitter, etc)

**Comment by tomsfernandez** [comment]
Hi @deiteris. This is a bug. The internal ticket https://www.mulesoft.org/jira/browse/APIMF-3026 was created to address this problem. All further communication will be in this Github issue. Thanks!

**Comment by deiteris** [comment]
The issue seems to be fixed with 5.0.5 as I'm no longer able to reproduce it. I'm closing this issue. Thanks!

**Comment by tomsfernandez** [comment]
Hi @deiteris, yes this issue was solved and released with the 5.0.5 release. Sorry for not updating this thread. Thanks for the proactivity closing this issue!.
